program/Cargo.toml

```diff
@@ -22,9 +22,9 @@ thiserror = "2.0"
 
 [dev-dependencies]
 lazy_static = "1.5.0"
+mollusk-svm = "0.0.13"
 proptest = "1.5"
 serial_test = "3.2.0"
-solana-program-test = "2.1.0"
```

Comment:
I can't believe dropping this single dependency sheds 4,143 lines from the lockfile. 😅